bindings/protocols/bacnet/bacnet-model.ttl

```diff
@@ -53,33 +53,50 @@ Additional parameters can be specified via URI variable syntax, e.g. ?bacnetCmdP
 
 bacnet:usesService rdf:type owl:DatatypeProperty ;
                    rdfs:domain hctl:Form ;
-                   rdfs:range [owl:oneOf ("ReadProperty"^^xsd:string "WriteProperty"^^xsd:string "SubscribeCOV"^^xsd:string "UnsubscribeCOV"^^xsd:string)] ;
+                   rdfs:range [owl:oneOf ("ReadProperty"^^xsd:string "WriteProperty"^^xsd:string "SubscribeCOV"^^xsd:string)] ;
                    rdfs:comment """
 Default BACnet service by td:OperationType:
 - td:readProperty -> ReadProperty
 - td:writeProperty -> WriteProperty
 - td:observeProperty -> SubscribeCOV, subscribe to object if no property ID is present in target, otherwise subscribe to property
-- td:unobserveProperty -> UnsubscribeCOV
+- td:unobserveProperty -> SubscribeCOV (will do an unsubscribe instead)
 
 TODO: Can we specify the defaults in a machine-readable, RDF-standard way?
 
 Not mapped yet:
-- td:invokeAction
+- td:invokeAction, td:queryAction, td:cancelaction
 - td:subscribeEvent, td:unsubscribeEvent
 - td:readAllProperties, td:writeAllProperties, td:readMultipleProperties, td:writeMultipleProperties, td:observeAllProperties, td:unobserveAllProperties
+- td:queryAllActions
+- td:subscribeallevents, td:unsubscribeallevents
```

Comment:
Do we want to use camelCase here as well?

Reply:
Yes we should: https://www.w3.org/2019/wot/td#unsubscribeAllEvents

Reply:
Thank you, fixed it now.